ereolen_support/ereolen_support.install

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * @file
+ * Prepare site for ereolen support module.
+ */
+
+/**
+ * Setup terms and vocabularies during install.
+ */
+function ereolen_support_install() {
+  if(empty(taxonomy_vocabulary_machine_name_load('ereolen_support_product'))) {
```

Comment:
Missing spaces between `if` and `(` for all if's in this function

Reply:
Added.